tests/PHPStan/Rules/Operators/data/invalid-inc-dec.php

```diff
@@ -15,4 +15,41 @@ function ($a, int $i, ?float $j, string $str, \stdClass $std) {
 	$j++;
 	$str++;
 	$std++;
+	$classWithToString = new ClassWithToString();
+	$classWithToString++;
+	$classWithToString = new ClassWithToString();
+	--$classWithToString;
```

Comment:
Bug: This does work in PHP 7: https://3v4l.org/oXvs8

Reply:
It actually works even without `__toString()`, but I'm not sure whether it can be used for anything useful with custom classes: https://3v4l.org/uXLrP I'd prefer to keep it disabled even for PHP 7 unless there is a use-case for it.